### PR TITLE
Multi index large

### PR DESCRIPTION
**Pull request by kimjh2005**
## Change Description

multi index smart contract for dogfooding test

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: examples/multi_index_large/include/multi_index_large.hpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+         uint64_t           u64 = 0;
+         uint128_t          u128 = 0ULL;
+         float              f64 = 0.0L;
+         long double        f128 = 0.0L;
+         eosio::checksum256 chk256;
+
+         uint64_t                  primary_key()const { return id; }
+         uint64_t                  get_u64()const     { return u64; }
+         uint128_t                 get_u128()const   { return u128; }
+         double                    get_f64()const     { return f64; }
```
Comment (linhuang-blockone) [inline examples/multi_index_large/include/multi_index_large.hpp:22]: not float?

File: examples/multi_index_large/include/multi_index_large.hpp [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      multi_index_large( name receiver, name code, datastream<const char*> ds )
+         : contract(receiver, code, ds), testtab(receiver, receiver.value) {}
+
+      struct [[eosio::table("data")]] main_record {
+         uint64_t           id         = 0;
+
+         uint64_t           u64 = 0;
+         uint128_t          u128 = 0ULL;
+         float              f64 = 0.0L;
+         long double        f128 = 0.0L;
```
Comment (linhuang-blockone) [inline examples/multi_index_large/include/multi_index_large.hpp:16]: not LL?

File: examples/multi_index_large/src/multi_index_large.cpp [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            u.u64 = u64;
+            u.u128 = u128;
+            u.f128 = f128;
+            u.chk256 = chk256;
+      });
+   }
+}
+
+[[eosio::action]] 
+void multi_index_large::print( uint64_t id ) {
```
Comment (linhuang-blockone) [inline examples/multi_index_large/src/multi_index_large.cpp:19]: why not pass in iter instead of id? The find can be saved.

**Review by linhuang-blockone — CHANGES_REQUESTED** [review]

File: examples/multi_index_large/include/multi_index_large.hpp (hunk 2 above)
Comment (kimjh2005) [reply examples/multi_index_large/include/multi_index_large.hpp:16]: L is Long double

File: examples/multi_index_large/src/multi_index_large.cpp (hunk 3 above)
Comment (kimjh2005) [reply examples/multi_index_large/src/multi_index_large.cpp:19]: It is action executed from cleos

File: examples/multi_index_large/include/multi_index_large.hpp (hunk 1 above)
Comment (kimjh2005) [reply examples/multi_index_large/include/multi_index_large.hpp:22]: changed to double abve.

**Review by linhuang-blockone — APPROVED** [review]

**Pull request merged**